docs/getting-started.rst

```diff
@@ -284,15 +284,3 @@ Start the docker container:
         -v /tmp/otel-collector-config.yaml:/etc/otel-collector-config.yaml \
         omnition/opentelemetry-collector-contrib:latest \
         --config=/etc/otel-collector-config.yaml
-
-Install the OpenTelemetry Collector exporter:
-
-.. code-block:: sh
-
-    pip install opentelemetry-ext-otcollector
-
-And execute the following script:
-
-.. literalinclude:: getting_started/otcollector_example.py
-    :language: python
-    :lines: 15-
```

Comment:
This is no longer needed?

Reply:
Yeah otcollector is now opencensus exporter or opentelemetry exporter, i think we need new getting started for either one of those (probably OT). I'll create a seperate issue to track this.

Reply:
Oh looks like @ocelotl  has already fixed this in his other PR!